tests/tests_common.py

```diff
@@ -263,54 +262,6 @@ def test_subscribe_events_to_wrong_node(self):
             handle = sub.subscribe_events(v)
         sub.delete()
 
-    def test_events_deprecated(self):
-        msclt = MySubHandlerDeprecated()
-        sub = self.opc.create_subscription(100, msclt)
-        handle = sub.subscribe_events()
-
-        ev = Event(self.srv.iserver.isession)
-        msg = b"this is my msg "
-        ev.Message.Text = msg
-        tid = datetime.utcnow()
-        ev.Time = tid
-        ev.Severity = 500
-        ev.trigger()
-
-        clthandle, ev = msclt.future.result()
-        self.assertIsNot(ev, None)  # we did not receive event
-        self.assertEqual(ev.Message.Text, msg)
-        #self.assertEqual(msclt.ev.Time, tid)
-        self.assertEqual(ev.Severity, 500)
-        self.assertEqual(ev.SourceNode, self.opc.get_server_node().nodeid)
-
-        # time.sleep(0.1)
-        sub.unsubscribe(handle)
-        sub.delete()
-
-    def test_events(self):
```

Comment:
I remove it because you have here generation of events and this can only server do... So the test with the client is not passing since there is no `get_event_object` function for client and it should not exist. Before we had EventGenerator so that passed.

for really test client to receive event, we should have client and server running at the same time, which I am not sure is possible here... If you have any ides you are welcome :)